packages/gatsby/content/getting-started/usage.md

```diff
@@ -65,6 +65,6 @@ yarn remove [package]
 ### Upgrading Yarn itself
 
 ```bash
-yarn set version 2
-yarn set version from sources
+yarn policies set-version berry
+yarn set version berry   
```

Comment:
```suggestion
yarn set version latest
yarn set version from sources
```
`yarn policies set-version` only works in v1.
`yarn set version from sources` is the correct command to use Yarn from master.

Reply:
I see @paul-soporan :D